pkgs/applications/office/gnucash/default.nix

```diff
@@ -56,6 +47,8 @@ stdenv.mkDerivation rec {
   # glib-2.62 deprecations
   NIX_CFLAGS_COMPILE = "-DGLIB_DISABLE_DEPRECATION_WARNINGS";
 
+  patches = [ ./0001-changes.patch ];
```

Comment:
I would add comment documenting why this patch is needed. I think the info you wrote to PR description would be sufficient.

Reply:
fixed in c1bd62d1e0a715f231f834b6295958ce38f52845